webpack.config.js

```diff
@@ -334,8 +334,13 @@ let appConfig = {
     ...(SHOULD_FORK_TS
       ? [
           new ForkTsCheckerWebpackPlugin({
-            eslint: TS_FORK_WITH_ESLINT,
-            tsconfig: path.resolve(__dirname, './config/tsconfig.build.json'),
+            eslint: {
+              enabled: TS_FORK_WITH_ESLINT,
+              files: './sentry/**/*.{ts,tsx,js,jsx}',
```

Comment:
this doesn't include test, also shouldn't it be `src/sentry`?

Reply:
😬 yes it should.

Does anyone use the eslint checker with this?

Reply:
@priscilawebdev does

Reply:
Looks like it won't actually run the older version of eslint we use on this newer version